doc/conf.py

```diff
@@ -169,7 +169,7 @@ def _custom_parse_generic_section(self, section, use_admonition):
     has_pyvista = any(['pyvista' in line for line in lines])
     if has_pyvista:
         for line in lines:
-            if '.plot(' in line or '.show(' in line:
+            if 'plot' in line or 'show' in line:
```

Comment:
Note to @adeak: This was done to allow methods like `plot_curvature` to generate plots.

Reply:
I figured, thanks.